compiler/rustc_borrowck/src/universal_regions.rs

```diff
@@ -688,7 +689,8 @@ impl<'cx, 'tcx> UniversalRegionsBuilder<'cx, 'tcx> {
         defining_ty: DefiningTy<'tcx>,
     ) -> ty::Binder<'tcx, &'tcx ty::List<Ty<'tcx>>> {
         let tcx = self.infcx.tcx;
-        match defining_ty {
+
+        let bound_ty = match defining_ty {
```

Comment:
```suggestion
        let inputs_and_output = match defining_ty {
```
it's a list of types